docker/ray/README.md

```diff
@@ -0,0 +1,20 @@
+## About
+Ray is an open source framework that provides a simple, universal API for building distributed applications. Ray is packaged with RLlib, a scalable reinforcement learning library, and Tune, a scalable hyperparameter tuning library. 
```

Comment:
I'd just link to the github page instead of writing an intro about ray here

Reply:
Done!